src/include/network/network_state.h

```diff
@@ -22,10 +22,7 @@ enum class ConnState {
   CONN_WAIT,       // State for waiting for some event to happen
   CONN_PROCESS,    // State that runs the network protocol on received data
   CONN_CLOSING,    // State for closing the client connection
-  CONN_CLOSED,     // State for closed connection
-  CONN_INVALID,    // Invalid STate
   CONN_GET_RESULT, // State when triggered by worker thread that completes the task.
```

Comment:
I am wondering if we may merge CONN_GET_RESULT and CONN_WAIT?

Reply:
Will do that in a separate PR to clean up all the states